### PR TITLE
Fix incorrect example ("Example  2")

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/web/html/media/audio.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/web/html/media/audio.md?ref=b5e14315f427da651895984050cd343d6935c254)

**Comment by kapnobatai137** [comment]
@Stefan-Stojanovic we should update the resource links as they don't work anymore

File: web/html/media/audio.md [hunk 1]
```diff
@@ -65,8 +65,8 @@ Example 2:
     type="audio/mpeg">
   <source src="some_song.ogg"
     type="audio/ogg">
-  <source src="some_song"
-    type="some_song.wav">
+  <source src="some_song.wav"
+    type="audio/mav">
```
Comment (kapnobatai137) [inline web/html/media/audio.md]: this should be `/wav` right?
```suggestion
    type="audio/wav">
```

Comment (Stefan-Stojanovic) [reply web/html/media/audio.md]: Yeah, accidentally put "m"

**Pull request merged**